code/datums/craft/recipes/alchemy.dm

```diff
@@ -83,189 +93,195 @@
-		list(/obj/item/stack/ore/uranium, 1),
+		list(/obj/item/stack/ore, 2), // Getting it from ore
 		list(QUALITY_HEATING, 5, 10)
 	)
 
-/datum/craft_recipe/alchemy/sacid
-	name = "Sulphuric Acid Bottle"
-	result = /obj/item/reagent_containers/glass/bottle/sacid
+/datum/craft_recipe/alchemy/carbon
+	name = "Carbon Bottle"
+	result = /obj/item/reagent_containers/glass/bottle/carbon
 	steps = list(
 		list(/obj/item/reagent_containers/glass, 1),
-		list(/obj/item/stack/ore, 1), //Leaching the sulfur form rocks
-		list(/obj/item/reagent_containers/food/snacks/egg , 1), //For good luck?
-		list(/obj/item/reagent_containers/food/snacks/meat/xenomeat , 1), //Used for base, has pacid in
+		list(/obj/item/reagent_containers/food/snacks/meat, 3), // Any meat will do
 		list(QUALITY_HEATING, 5, 10)
 	)
 
-/datum/craft_recipe/alchemy/sacid
-	name = "Sulphuric Acid Bottle"
-	result = /obj/item/reagent_containers/glass/bottle/sacid
+/datum/craft_recipe/alchemy/copper
+	name = "Copper Bottle"
+	result = /obj/item/reagent_containers/glass/bottle/copper
 	steps = list(
 		list(/obj/item/reagent_containers/glass, 1),
-		list(/obj/item/stack/ore/glass, 1),
+		list(/obj/item/stack/cable_coil, 30), // Cables have copper!
 		list(QUALITY_HEATING, 5, 10)
 	)
 
-/datum/craft_recipe/alchemy/sodium
-	name = "Sodium Bottle"
-	result = /obj/item/reagent_containers/glass/bottle/sodium
+/datum/craft_recipe/alchemy/ethanol
+	name = "Ethanol Bottle"
+	result = /obj/item/reagent_containers/glass/bottle/ethanol
 	steps = list(
 		list(/obj/item/reagent_containers/glass, 1),
-		list(/obj/item/reagent_containers/food/snacks/meat, 3), //Not much but hey its something
+		list(/obj/item/reagent_containers/food/snacks/grown, 3),
 		list(QUALITY_HEATING, 5, 10)
 	)
 
-/datum/craft_recipe/alchemy/potassium
-	name = "Potassium Bottle"
-	result = /obj/item/reagent_containers/glass/bottle/potassium
+/datum/craft_recipe/alchemy/hclacid
+	name = "Hydrochloric Acid Bottle"
+	result = /obj/item/reagent_containers/glass/bottle/hclacid
 	steps = list(
 		list(/obj/item/reagent_containers/glass, 1),
-		list(/obj/item/reagent_containers/food/snacks/grown, 3),
+		list(/obj/item/stack/ore, 1),
+		list(/obj/item/stack/ore/iron, 1), // Two steps to get sulfuric acid...
+		list(/obj/item/stack/ore, 1), // ...And make them react with rock salt
 		list(QUALITY_HEATING, 5, 10)
 	)
 
-/datum/craft_recipe/alchemy/phosphorus
-	name = "Potassium Bottle"
-	result = /obj/item/reagent_containers/glass/bottle/phosphorus
+/datum/craft_recipe/alchemy/hydrazine
+	name = "Hydrazine Bottle"
+	result = /obj/item/reagent_containers/glass/bottle/hydrazine
 	steps = list(
 		list(/obj/item/reagent_containers/glass, 1),
-		list(/obj/item/reagent_containers/food/snacks/grown, 3),
-		list(/obj/item/reagent_containers/food/snacks/meat, 3),
+		list(/obj/item/reagent_containers/food/snacks/grown, 1), // Hydrogen from biomass
+		list(/obj/plant_spawner/grass, 1), // Nitrogen from plants
```

Comment:
this is not something thats can be gotton by players

Reply:
You can literally go outside and use a cutting tool on any grass object on the jungle, even the colony upper level, and get those small plant spawner grass things though

Reply:
Plant spawner, dosnt stick around, it makes a grown fruit of grass on init